src/programs/Welcome.tsx

```
@@ -74,140 +43,179 @@ const Welcome = ({ id }: props) => {
-              <p className={styles.subtitle}>{INTERESTS2}</p>
-              <br></br>
-              <p className={styles.subtitle}>{INTERESTS3}</p>
+          </div>
+        </WinAccordion>
+      </div>
+
+      <div className={styles.rightpanel}>
+        {!aboutmeView ? (
+          <div style={{ width: "100%" }}>
+            <h3 className={styles.welcome_text}>
+              Welcome To Vincenzo Reina's GTM Hub
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)